README.md

```diff
@@ -1,40 +1,40 @@
-Riot
-====
+Element
+=======
 
-Riot (formerly known as Vector) is a Matrix web client built using the [Matrix
+Riot (formerly known as Vector and Riot) is a Matrix web client built using the [Matrix
 React SDK](https://github.com/matrix-org/matrix-react-sdk).
 
 Supported Environments
 ======================
 
-Riot has several tiers of support for different environments:
+Element has several tiers of support for different environments:
 
 * Supported
   * Definition: Issues **actively triaged**, regressions **block** the release
   * Last 2 major versions of Chrome, Firefox, and Safari on desktop OSes
-  * Latest release of official Riot Desktop app on desktop OSes
+  * Latest release of official Element Desktop app on desktop OSes
   * Desktop OSes means macOS, Windows, and Linux versions for desktop devices
     that are actively supported by the OS vendor and receive security updates
 * Experimental
   * Definition: Issues **accepted**, regressions **do not block** the release
-  * Riot as an installed PWA via current stable version of Chrome, Firefox, and Safari
+  * Element as an installed PWA via current stable version of Chrome, Firefox, and Safari
   * Mobile web for current stable version of Chrome, Firefox, and Safari on Android, iOS, and iPadOS
 * Not supported
   * Definition: Issues only affecting unsupported environments are **closed**
   * Everything else
 
-For accessing Riot on an Android or iOS device, we currently recommend the
+For accessing Element on an Android or iOS device, we currently recommend the
 native apps [riot-android](https://github.com/vector-im/riot-android)
 and [riot-ios](https://github.com/vector-im/riot-ios).
 
 Getting Started
 ===============
 
-The easiest way to test Riot is to just use the hosted copy at https://riot.im/app.
-The `develop` branch is continuously deployed by Jenkins at https://riot.im/develop
+The easiest way to test Element is to just use the hosted copy at https://app.element.io.
+The `develop` branch is continuously deployed by Jenkins at https://develop.element.io
```

Comment:
I had to guess the develop link. Is it alright?

Reply:
Yes, that's correct. 😄

Reply:
Noice